src/ProjectConfiguration.ts

```diff
@@ -16,6 +16,12 @@ export enum AppType {
   SERVICE = 'service',
 }
 
+export interface Tile {
+  name: string;
+  uuid: string;
+  buildTargets?: string[]; // If not specified, defaults to all buildTargets
```

Comment:
nit: I think if you turn this into form `/* */` and place on the line above this will better appear in auto-complete context

Reply:
Good idea. Making the changes.